packages/mobile/src/page/Cafeteria/index.tsx

```diff
@@ -1,7 +1,71 @@
+import { Link } from "react-router-dom";
+
+import classnames from "classnames";
+import { cafeteriaMenu } from "src/__mocks__";
+import noMenu from "src/assets/no_menu.png";
+import { Arrow } from "src/components/atoms/icon/Arrow";
+import CafeteriaMenuCard from "src/components/molecules/CafeteriaMenuCard";
+import Footer from "src/components/molecules/Footer";
+import useFlicking from "src/hooks/useFlicking";
+import { cafeteriaTime } from "src/utils/cafeteriaTime";
+
 import $ from "./style.module.scss";
 
+const menuList = [
+  { id: 1, name: "본관" },
+  { id: 2, name: "양성재" },
+  { id: 3, name: "양진재" },
+  { id: 4, name: "한빛식당" },
+  { id: 5, name: "별빛식당" },
+  { id: 6, name: "은하수식당" },
+];
+
 function Cafeteria() {
-  return <section className={$.cafeteria}>Cafeteria</section>;
+  const [ clickedMenu, Flicking ] = useFlicking(menuList);
+
+  return (
+    <>
+      {Flicking}
+      <main
+        className={classnames($.cafeteria, {
+          [$["no-menu"]]: !cafeteriaMenu.length,
+        })}
+      >
+        {cafeteriaMenu.length > 0 ? (
+          <>
+            {cafeteriaMenu.map(({ id, content, calory, time }) => {
+              const [ mealTime, timeInfo ] = cafeteriaTime(id, time);
+              return (
+                <CafeteriaMenuCard
+                  key={content}
+                  {...{ mealTime, timeInfo }}
+                  mealMenu={content}
+                  calory={calory}
```

Comment:
위에서 써주신 전개 연산자에 `calory`도 포함할 수 있을 것 같습니다~
```suggestion
                <CafeteriaMenuCard
                  key={content}
                  {...{ mealTime, timeInfo, calory }}
                  mealMenu={content}
```